csharp/dotnet/security/use_deprecated_cipher_algorithm.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+  - id: use_deprecated_cipher_algorithm
+    patterns:
+      - pattern: $KEYTYPE.Create(...);
+      - metavariable-pattern:
+          metavariable: $KEYTYPE
+          pattern-either:
+            - pattern: DES
+            - pattern: RC2
+    message: Usage of deprecated cipher algorithm detected
```

Comment:
Please include a recommendation for which algorithm to use instead (preferably with mention of the exact API to use). E.g.

> Use the AES algorithm instead. This can be done with the `Aes.create()` function.

Reply:
62088b5a9baa5943821bb58cc7c93c8991680668